engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -1496,6 +1501,9 @@ static void LogFrameBufferError(GLenum status)
 
             InitializeSetTextureAsyncState(context->m_SetTextureAsyncState);
 
+            context->m_GLHandlesData.m_Mutex = dmMutex::New();
+            context->m_AssetHandleContainerMutex = dmMutex::New();
```

Comment:
I _think_ we want two different mutexes here, as they protect different things 🤔